lib/postgrex.ex

```diff
@@ -65,6 +65,23 @@ defmodule Postgrex do
   `Postgrex` uses the `DBConnection` framework and supports all `DBConnection`
   options like `:idle`, `:after_connect` etc.
   See `DBConnection.start_link/2` for more information.
+
+  ## Examples
+
+      iex> {:ok, pid} = Postgrex.start_link(database: "postgres")
+      {:ok, #PID<0.69.0>}
+
+  Run a query after connection has been established:
+
+      defmodule MyMod do
+        def set_timezone(pid) do
+          {:ok, _} = Postgrex.query(pid, "SET TIME ZONE 'UTC';", [])
+        end
+      end
+
```

Comment:
Could we simplify this to `&Postgrex.query!(&1, "SET TIME ZONE 'UTC';", [])` so that the example is a simple as psosible. Note also that it is not `pid` in an after_connect.